web/src/components/pages/contribution/contribution.tsx

```diff
@@ -242,11 +255,11 @@ class ContributionPage extends React.Component<Props, State> {
       onSubmit,
       type,
     } = this.props;
-
     if (
       event.ctrlKey ||
       event.altKey ||
       event.shiftKey ||
+      event.metaKey ||
```

Comment:
I noticed that using `cmd-r` to refresh on mac was triggering a record instead of a refresh.